sdk-core/transaction.md

```diff
@@ -44,6 +44,17 @@ dto DraftTransaction:
 ```
 class TransactionComputer:
     compute_transaction_fee(transaction: Transaction, network_config: INetworkConfig): Amount;
+
     compute_bytes_for_signing(transaction: Transaction): bytes;
+
     compute_transaction_hash(transaction: Transaction): bytes;
+
+    // checks if the second least significant bit is set; returns True if it's set, False otherwise
```

Comment:
`Returns whether ... of "transaction.options" is set`.

Also, since "guarded" might refer to the presence of the `guardian` and `guardianSignature` fields when we use the word in real-life (the word is somehow ambiguous for us), perhaps rename to:

```
has_transaction_option_guarded
```

? (opinion)

Reply:
On this matter, perhaps also add `has_transaction_option_hash_signing` or something similar?

For reference: https://github.com/multiversx/mx-sdk-js-core/blob/main/src/constants.ts#L3C34-L3C44.

Reply:
Now we have two methods `has_options_set_for_guarded_transaction` and `has_options_set_for_hash_signing`.